packages/lesswrong/components/review/ReviewVotingCanvas.tsx

```diff
@@ -165,6 +165,23 @@ const styles = (theme: ThemeType) => ({
     maxWidth: '1000px',
     marginLeft: '-22px'
   },
+  callToAction: {
+    ...theme.typography.headerStyle,
+    fontWeight: 700,
+    textAlign: 'center',
+    fontSize: 30,
+    width: 'calc(100vw - 1250px)',
+    minWidth: 200,
+    zIndex: 1,
+    position: 'absolute',
+    top: 230,
+    left: 10,
+    textShadow: '0 0 5px rgba(255, 255, 255, 1), 0 0 10px rgba(255, 255, 255, 1), 0 0 15px rgba(255, 255, 255, 1), 0 0 20px rgba(255, 255, 255, 1)',
```

Comment:
These need to be using `theme.palette`, since we disallow hard-coded RGB values in styles

Reply:
(the parts that need to be replaced are the instances of `rgba(255, 255, 255, 1)`; I think we have something for this but don't remember thing.